titan-web-client/src/components/core/TitanApp.js

```diff
@@ -77,27 +76,25 @@ class TitanApp extends React.Component {
 
   render () {
     return (
-      <ThemeProvider theme={defaultTheme}>

```

Comment:
We're using two theme providers: one from `styled-components` and another from `material-ui`. The material UI theme provider allows us to override component styles and color schemes, where as the styled components provider doesn't offer any additional value.

The providers were conflicting and we really only need one, so I removed the styled components provider.